app/res/values/themes.xml

```diff
@@ -119,8 +120,18 @@
     <style name="TextAppearance.Custom.WindowTitle.Case" parent="TextAppearance.Custom.WindowTitle">
     </style>
 
-    <style name="commcare_button_style" parent="Widget.AppCompat.Button">
+    <style name="Commcare.Button.Primary" parent="Widget.AppCompat.Button">
         <item name="textAllCaps">false</item>
+        <item name="android:minHeight">60dp</item>
+        <item name="android:textColor">@color/white</item>
+        <item name="backgroundTint">@color/primary_button_background</item>
+        <item name="cornerRadius">@dimen/dp6</item>
+    </style>
+
+    <style name="CommCare.Button.Default" parent="Widget.AppCompat.Button">
+        <item name="textAllCaps">false</item>
+        <item name="backgroundTint">@null</item>
+        <item name="android:textColor">@android:color/primary_text_light</item>
     </style>
 
     <style name="TextAppearance.Custom.WindowTitle.Menu" parent="TextAppearance.Custom.WindowTitle">

```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/5a959df2d1681e0a640ebb72d4513eaecefa585e/app/res/values/themes.xml#L137"></span>The resource <code>R.style.TextAppearance_Custom_WindowTitle_Menu</code> appears to be unused</td>
    </tr>
  </tbody>
</table>